stdlib/gc.mli

```diff
@@ -449,7 +433,6 @@ val delete_alarm : alarm -> unit
 (** [delete_alarm a] will stop the calls to the function associated
    to [a]. Calling [delete_alarm a] again has no effect. *)
 
-<<<<<<< HEAD
```

Comment:
Why keep our code here? Presumably for compatibility with the ocaml 4 runtime. But then we should probably leave a `CR ocaml 5 runtime` (?) or something to come back and fix this later.

Reply:
Sounds good to me.